uffizzi/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Build
+
+FROM golang:1.18 as build 
+WORKDIR /go/src/github.com/jesseduffield/lazygit/
+COPY go.mod go.sum ./
+RUN go mod download
+COPY . .
+RUN CGO_ENABLED=0 GOOS=linux go build
+
+FROM alpine:3.15
+RUN apk add --no-cache -U git xdg-utils
+WORKDIR /go/src/github.com/jesseduffield/lazygit/
+COPY --from=build /go/src/github.com/jesseduffield/lazygit ./
+COPY --from=build /go/src/github.com/jesseduffield/lazygit/lazygit /bin/
```

Comment:
This section looks redundant and doesn't get used?

Reply:
Fixed, with new docker image

Reply:
Right, fixed.